diff --git a/node_modules/@openzeppelin/contracts/utils/Context.sol b/node_modules/@openzeppelin/contracts/utils/Context.sol
index 3d07c4a..8183326 100644
--- a/node_modules/@openzeppelin/contracts/utils/Context.sol
+++ b/node_modules/@openzeppelin/contracts/utils/Context.sol
@@ -17,7 +17,7 @@ abstract contract Context {
         return msg.sender;
     }
 
-    function _msgData() internal view virtual returns (bytes memory) {
+    function _msgData() internal view virtual returns (bytes calldata) {
         this; // silence state mutability warning without generating bytecode - see https://github.com/ethereum/solidity/issues/2691
         return msg.data;
     }
